src/RDWOpenData/RDW.php

```diff
@@ -56,7 +56,7 @@ protected static function mergeOutput(array $data, array $output, string $endpoi
 				}
 			}
 		} else {
-			$data = array_merge($data, $output[0]);
+			$data = array_merge($data, $output[0]) ?? [];
```

Comment:
please put ?? [] inside of array_merge

Reply:
This doesn't guarantee that `array_merge` returns a non-null value, which is necessary to comply with the function signature